long-live-pomf.md

```diff
@@ -3,7 +3,7 @@
 | https://mixtape.moe/       | Up               | https://gitgud.io/Sapphire/mixtape.moe        |              100 | Pastebin, voice
 | https://safe.moe/          | Up               | https://github.com/WeebDev/loli-safe          |              200 | Pastebin
 | https://sub.god.jp/        | Up               | https://github.com/xatasan/registrars         |               32 | JS-free, pastebin
-| https://plexle.fi          | Up               | https://github.com/pomf/pomf                  |               20 | Zero Logs.               
+| https://l.shenron.su       | Up               | https://github.com/pomf/pomf                  |               20 | Zero Logs.               
```

Comment:
The `source code` column is meant to link to the source code of the particular instance (with all customizations etc), not the codebase it was based on.